front/app/containers/Authentication/steps/Onboarding/TopicsAndAreas/index.tsx

```diff
@@ -18,8 +18,17 @@ interface Props {
 const TopicsAndAreas = ({ onSubmit, onSkip }: Props) => {
   const { formatMessage } = useIntl();
   const { data: authUser } = useAuthUser();
-  const { data: areas } = useAreas({ forOnboarding: true });
-  const { data: topics } = useTopics({ forOnboarding: true });
+
+  const { data: areas } = useAreas({
+    forOnboarding: true,
+    sort: 'projects_count',
+  });
```

Comment:
This way, the request is the same as the one in `app/components/Areas/index.tsx`. Since they're the same, they will be cached, and we only need to make the request once. Same for `useTopics` below